src/test/java/com/global/api/tests/realex/RealexCreditTests.java

```diff
@@ -212,6 +212,71 @@ public void creditAuthorization_WithMultiAutoSettle() throws ApiException {
         assertNotNull(response);
         assertEquals("00", response.getResponseCode());
     }
+    
+    @Test
+    public void creditAuthorization_WithMultiAutoSettle_WithEstNumTxn() throws ApiException {
+        Transaction response = card.authorize(new BigDecimal("14"))
+                .withEstimatedTransactions(3)
+                .withCurrency("USD")
+                .withMultiCapture(true)
+                .execute();
+        assertNotNull(response);
+        assertEquals("00", response.getResponseCode());
+    }
+    
+    @Test
+    public void creditMultisettle() throws ApiException {
+        Transaction response = card.authorize(new BigDecimal("17"))
+                .withEstimatedTransactions(3)
+                .withCurrency("USD")
+                .withMultiCapture(true)
+                .execute();
+        assertNotNull(response);
+        assertEquals("00", response.getResponseCode());
+
+        Transaction rebate = response.multicapture(new BigDecimal("17"))
+                .withAmount(new BigDecimal("5.00"))
+                .withCurrency("USD")
+                .execute();
+        assertNotNull(rebate);
+        assertEquals("00", rebate.getResponseCode());
+    }
+    
+    @Test
+    public void creditMultisettle_WithEstNumTxn() throws ApiException {
+        Transaction response = card.authorize(new BigDecimal("17"))
+                .withCurrency("USD")
+                .withMultiCapture(true)
+                .execute();
+        assertNotNull(response);
+        assertEquals("00", response.getResponseCode());
+
+        Transaction rebate = response.multicapture(new BigDecimal("17"))
+                .withEstimatedTransactions(3)
+                .withAmount(new BigDecimal("5.00"))
+                .withCurrency("USD")
+                .execute();
+        assertNotNull(rebate);
+        assertEquals("00", rebate.getResponseCode());
+    }
+    
+    @Test
+    public void creditMultisettle_WithFinalFlag() throws ApiException {
+        Transaction response = card.authorize(new BigDecimal("17"))
+                .withCurrency("USD")
+                .withMultiCapture(true)
+                .execute();
+        assertNotNull(response);
+        assertEquals("00", response.getResponseCode());
+
+        Transaction rebate = response.multicapture(new BigDecimal("17"))
+                .withAmount(new BigDecimal("5.00"))
```

Comment:
These calls are confusing to developers reviewing these tests. It appears that the initial authorization amount needs to be set as well as the capture amount, but all this is doing is setting the amount to `17` and then to `5`. In another way, here's the same two lines written slightly different:

```java
Transaction capture = response.multicapture()
    .withAmount(new BigDecimal("17"))
    .withAmount(new BigDecimal("5"))
```

Let's change this (and other tests) to only pass in the single, required amount for the capture, e.g.:

```java
Transaction capture = response.multicapture(new BigDecimal("5.00"))
```

Reply:
Late night copy and paste... I will fix that.